sample-demo-flask-app/generate-traffic.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+START_DATETIME=$(date +%s)
+
+COUNT=1000
+
+for i in $(seq 1 $COUNT); do
+  echo "> Running #$i from ${COUNT}... "
+  sleep 1
+  pytest integration_tests
```

Comment:
Shouldnt curl be sufficient to generate traffic? What is running the integration test's work here?

Reply:
I'm trying to diversify the traffic, that's why I added something as integration test, to have a get, post, more frames.